### DISCUSSION TITLE
How to comment out Text?

### DISCUSSION THREAD

**Discussion by JulianHSC** [microsoft/react-native-windows#12948] [Q&A] [answered]
I tried to add or change the code in the training development website: https://reactnative.dev/docs/intro-react?language=typescript. When I tried to put /* and */ for multiline comments, my react native run command keeps showing an error. I just tried this and it won't work. Update: I'm able to comment out the onPress function.
       /*
      <Text>Hello, I am your cat!</Text>
      */

OR

      <Text>/* Hello, I am your cat! */</Text>
Is there a way to insert multiline comments outside so I can keep the code without using it? This is for my own reference.

**Comment by acoates-ms** [comment, marked as answer]
I think this is what you want.

```jsx
  <Text>{/* Hello, I am your cat! */}</Text>
```

**Comment by JulianHSC** [comment]
Yes. I just found a video where curly braces it outside the text tag. That can also be done.  Why is "/* " and " */" not enough? What's different about react native? Can someone answer?

**Reply by chrisglein** [reply]
This repo is specifically for questions about React Native for Windows. Your question here is about React Native generically, or really even just React and JSX. You'll get the best results if you look within that audience (there are a _ton_ more resources).

A simple search for "React Native how to comment out Text" produced many viable results. For example, this: https://stackoverflow.com/questions/30766441/how-to-use-comments-in-react

**Comment by JulianHSC** [comment]
It's all for React native for mobile programming. Is this the correct place to ask?

**Reply by chrisglein** [reply]
No. This is the React Native for Windows repo, for development of React Native apps for desktop, not mobile.
